stock_warehouse_orderpoint_stock_info/models/stock_warehouse_orderpoint.py

```diff
@@ -7,21 +7,29 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 from odoo import api, fields, models
+from collections import defaultdict
 
 
 class StockWarehouseOrderpoint(models.Model):
     _inherit = 'stock.warehouse.orderpoint'
 
     @api.multi
     def _compute_product_available_qty(self):
-        for rec in self:
-            product_available = rec.product_id.with_context(
-                location=rec.location_id.id
-                )._product_available()[rec.product_id.id]
-            rec.product_location_qty = product_available['qty_available']
-            rec.incoming_location_qty = product_available['incoming_qty']
-            rec.outgoing_location_qty = product_available['outgoing_qty']
-            rec.virtual_location_qty = product_available['virtual_available']
+        op_by_loc = defaultdict(lambda: self.env['stock.warehouse.orderpoint'])
+        for order in self:
+            op_by_loc[order.location_id] |= order
+        for location_id, order_in_loc in op_by_loc.items():
+            products = order_in_loc.mapped('product_id').with_context(
+                location=location_id.id)._compute_quantities_dict(
+                        lot_id=self.env.context.get('lot_id'), 
+                        owner_id=self.env.context.get('lowner_id'), 
+                        package_id=self.env.context.get('lpackage_id'))
```

Comment:
'lowner_id' → 'owner_id'
'lpackage_id' → 'package_id'

But as @rousseldenis said, maybe is not needed these things.